k9mail/src/main/java/com/fsck/k9/message/html/HtmlSanitizer.java

```diff
@@ -24,6 +24,10 @@
                 .addAttributes("td",
                         "align", "bgcolor", "colspan", "headers", "height", "nowrap", "rowspan", "scope", "valign",
                         "width")
+                .addAttributes("map", "name")
+                .addAttributes("area", "shape", "coords", "href", "alt")
```

Comment:
Looks like there are a couple of other attributes we should add as well. See https://www.w3schools.com/tags/tag_area.asp

Reply:
Yeah, I did look at I wasn't sure we actually wanted them all. Certainly I don't really want `target` for example. I'll have a look what gets allowed for `a` and add similar ones.

Reply:
Apparently we only allow `"href", "title"` for `a`. So I think this is sufficient.